templates/cdap-deployment.yaml

```diff
@@ -106,6 +106,10 @@ spec:
             - name: cdap-cm-vol-{{$k}}
               mountPath: {{$v}}
             {{end}}
+            {{range $k,$v := $.Base.SecretVolumes}}
+            - name: cdap-secret-vol-{{$k}}
```

Comment:
Consider shortening it to cdap-se-vol-

Recall I ran into name string length limit issue when calling it cdap-configmap-vol-

Reply:
The name limit is 253 characters. Did we run issue before?